src/components/header.js

```diff
@@ -64,7 +72,11 @@ function Header() {
               title: `Code Of Conduct`,
             },
           ].map(link => (
```

Comment:
I'm happy to find a map function here. I've never seen it attached to the end of a bracket. I'm interested to see how it's working here.

Reply:
It's pretty cool! I believe that's something that you can do in any ES6 file: Take an array and attach a function directly to the end.